.github/workflows/code-checking.yml

```diff
@@ -20,3 +20,42 @@ jobs:
         run: npm run prettier --if-present
       - name: Run linter
         run: npm run lint --if-present
+
+  version-check:
+    name: Run version Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Check version
+        uses: SADEInnovations/version-checker@master
+        id: checker
+        env:
+          INPUT_FILE: "package.json"
+      - name: Results
+        run: echo "Version is ${{ steps.checker.outputs.version }}"
+
+  PR-title-checking:
+    name: Run PR Title Check
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    steps:
+      - uses: deepakputhraya/action-pr-title@master
+        with:
+          regex: '(^\[\w+-\d+|CHORE|HAR\]\s[\w\d].+$)'
```

Comment:
regex does not work properly. I changed the title to "[HAR-10 more workflows" (missing closing parenthesis) and it still validates it